scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/http/ResponseSerializers.java

```diff
@@ -95,7 +95,7 @@ public static JsonValue serializeScheduleResults(final ScheduleResults results)
             .entrySet()
             .stream()
             .collect(
-                Collectors.toMap(e -> Long.toString(e.getKey().id()), Map.Entry::getValue)));
+                Collectors.toMap(e -> Long.toString(e.getKey().goalInvocationId().get()), Map.Entry::getValue)));
```

Comment:
Future work discussed with @skovati: `GoalId` should be split into a pair of `GoalInvocationId` and `GoalDefinitionId` (or something similarly named). This will avoid us needing to use `Optional` to meet the one use case of updating a goal definition's parameter schema.

Reply:
[Issue for this work](https://github.com/NASA-AMMOS/aerie/issues/1551)